docs/UserGuide.md

```diff
@@ -209,6 +214,43 @@ Tags: SevereAllergies
 You now have 1 patient(s) in your contact book.
 ```
 
+### Changing remarks of a patient : `remark` 
+Adds a remark to the specified patient in the address book.
+
+Format: `remark INDEX r\REMARK`
+
+- Replaces any existing remarks for the patient at the specified `INDEX` with `REMARK`
+- The index refers to the index number shown in the displayed list.
+- The index **must be a positive integer** 1, 2, 3,...
```

Comment:
Can we make the formatting of the bullet points to be consistent with the whole document? We are following * instead of - for bullet points